src/schema/automobile-insurance.json

```diff
@@ -0,0 +1,139 @@
+            "description": "Emitted when actual cover information is collected.<br/>Note: on deep links with pre-selected additional cover this output may not be provided.",
+            "experimental": true
+        },
+        "availablePersonalInjuryCovers": {
+            "typeRef": "#/domains/FlightBooking/types/AvailableCovers",
+            "description": "Emitted when actual cover information is collected.<br/>Note: on deep links with pre-selected additional cover this output may not be provided.",
+            "experimental": true
+        }
+    },
+    "types": {
+        "PurchaseConfirmation": {
+            "type": "object",
+            "description": "Information gathered on the \"confirmation\" page.",
+            "properties": {
+                "purchaseReference": {
+                    "type": "string",
+                    "description": "Reference code grabbed from the page.",
+                    "example": "RL2XYZ"
+                },
+                "message": {
+                    "type": "string",
+                    "description": "The confirmation message.",
+                    "example": "Thank you for using UBIO Insurance. Your quote is ABC123 and purchase reference is DEF456."
+                },
+                "price": {
+                    "$ref": "#/domains/Generic/types/Price",
+                    "description": "Price, if found on the confirmation page."
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "purchaseReference"
+            ]
+        },
+        "Options": {
+            "type": "object",
+            "description": "Flags for enabling optional automation features such as upsells and additional cover. Clients should send <code>false</code> or omit flags if specific features are not implemented on their end.",
+            "experimental": true,
+            "properties": {
+                "legalCover": {
+                    "type": "boolean",
+                    "default": false
+                },
+                "breakdownCover": {
+                    "type": "boolean",
+                    "default": false
+                },
+                "personalInjuryCover": {
+                    "type": "boolean",
+                    "default": false
+                }
+            }
+        },
+        "AvailableCovers": {
+            "type": "array",
+            "experimental": true,
+            "description": "A list of available cover options found on the page.",
+            "minItems": 1,
+            "maxItems": 9,
+            "items": { "$ref": "#/domains/AutomobileInsurance/types/Cover" }
+        },
+        "Cover": {
+            "type": "object",
+            "experimental": true,
+            "description": "Flight fare information.",
+            "additionalProperties": false,
+            "properties": {
+                "name": {
```

Comment:
How does one "skip" - i.e. select none of the options? I had it as an extra option as `skip:true`